.bazelci/presubmit.yml

```diff
@@ -0,0 +1,42 @@
+---
+platforms:
+  ubuntu1804:
+    build_flags:
+    - "--build_tag_filters=-nolinux"
```

Comment:
Let's remove these filters. We don't have any platform restrictions right now.

Reply:
Done